ConnectedVessels5_4.py

```diff
@@ -0,0 +1,9 @@
+def interleave(*parameters):
+  listReturn=[]
+  for i,element in enumerate(parameters):
+    for param in parameters:
+      listReturn+=[param[i]]
+  return listReturn
+
+
+#interleave('abc', [1, 2, 3], ('!', '@', '#'))
```

Comment:
why is this comment out and not in https://www.freecodecamp.org/news/if-name-main-python-example/